### ISSUE TITLE
Try to consistently fix keybindings in iTerm2

### ISSUE THREAD

**Issue by raxod502** [radian-software/radian#101]
As anyone who uses Emacs through a terminal emulator knows, support for keybindings in terminals sucks. A lot. You have to do a lot of hackery to get even simple things like `C-)` to work! It would be nice to have a feature in the setup script for installing iTerm2 and fixing its keybindings to avoid as many of these problems as possible. We might have to do some tmux unbreaking as well.

**Comment by raxod502** [comment]
Blocked by #113.

**Comment by raxod502** [comment]
Closing as stale: I no longer use Emacs in the terminal except very occasionally, and nobody else has requested this feature.
